students/tim_lurvey/lesson03/mailroom_pt1.py

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env python
+__author__ = 'Tim Lurvey, ig408c'
+
+import sys
+
+sys.path.append('C:\\Users\\ig408c\\Documents\\!SCHOOL\\!Python\\SP_Online_PY210-master\\students\\tim_lurvey')
+from myClasses import bunch

```

Comment:
Typically want your import statements together at the top of the file

Reply:
Sorry, I had hard coded the sys path to myClasses.  I have the file in my /tim_lurvey path and my next pull will include the __init__.py to allow the code to be executed.  The class it really just a fancy dictionary, but it makes code easier to read.